### ISSUE TITLE
Group FDR control

### ISSUE THREAD

**Issue by JWJW1** [Nesvilab/FragPipe#1922]
<img width="925" alt="스크린샷 2024-12-09 오후 8 41 03" src="https://github.com/user-attachments/assets/db63f3ff-8cd9-4114-aa98-2bedf5f838ab">
I would like to make group fdr control different from Fragpipe to PSM level 1% and 5%. I can't find a way to proceed with FDR control at a different rate, so I would appreciate it if you could tell me if there is a way or how I can get the data as the next best way.
Another question. --quantitative --prot 1 --group --psm 0.01 --ion 1 --pep 1 As a result of running Fragpipe like this, the file psm.tsv file has FDR control based on what column (q-value, probability, score(D/T))...?
Lastly, when I send and run the validation tab with the image, "Probability" is created, which is a column related to peptide-prophet, why?

**Comment by JWJW1** [comment]
Yes, thank you for your answer. I read the tutorial. I would like to give PSM level FDR control to 0.01 and 0.05 for each group. How can I set the parameters?

**Comment by JWJW1** [comment]
Thank you for your answer. I have a question about that. I'm trying to do group FDR with group1 at FDR1% and group2 at FDR 5%, so how can I enter the parameter on Fragpipe? Or is there another way?

**Comment by fcyu** [comment]
It is not supported. The groups can only have the same FDR level.

Best,

Fengchao

**Comment by JWJW1** [comment]
Thank you for your answer. Then I would like to refer to the standards that Fragpipe performed FDR filtering on. Is there a standard that Fragpipe (philosopher) can control FDR at the PSM level? For example, if you calculate with Decoy/Target, you have to sort it with that column, is there a standard column?

**Comment by Amelia811** [comment]
I also have questions about validation tab. What does it mean as default setting in PSM validation run percolator min probability 0.5？ 
Where can I set the PSM，peptide and protein FDR cutoff？
Is there any tutorials about this in detail？
Thanks.

**Comment by fcyu** [comment]
> Is there a standard that Fragpipe (philosopher) can control FDR at the PSM level? For example, if you calculate with Decoy/Target, you have to sort it with that column, is there a standard column?

Yes, it is the `Probability` columns that is used to calculate the decoy target ratios.

Best,

Fengchao

**Comment by fcyu** [comment]
> What does it mean as default setting in PSM validation run percolator min probability 0.5？

All PSMs with Percolator probability < 0.5 will be discarded and not used in the downstream steps.

> Where can I set the PSM，peptide and protein FDR cutoff？

Hovering your mouse over the "filter" textbox will show 
![image](https://github.com/user-attachments/assets/c57ef9d9-b916-4e9b-b83e-d206056bf218)

Best,

Fengchao

**Comment by Amelia811** [comment]
Thank you for your reply. If I set the filter tab as below, is the PSM and peptide FDR is controlled <1% by default?
![1733967795439](https://github.com/user-attachments/assets/8b2f1cd9-2c32-4c04-b9c3-45b64f0d0afe)

**Comment by fcyu** [comment]
Yes.

Best,

Fengchao

**Comment by Amelia811** [comment]
If I delete "--prot 0.01" but keep "--sequential --picked", is the PSM, peptide and protein FDR are all controlled <1% by default?
Thanks again.

**Comment by fcyu** [comment]
Yes.

Best,

Fengchao
